digitaltwin_client/readme.md

```diff
@@ -8,6 +8,8 @@ These instructions and samples assume basic familiarity with Digital Twin concep
 ### Building
 Instructions on how to build the Digital Twin Client SDK or else retrieve packages for it are available [here](./doc/building_sdk.md).  If you are familiar with building the IoTHub Device SDK, the instructions are almost identical.
 
+**Please use [public-preview](https://github.com/Azure/azure-iot-sdk-c/tree/public-preview) branch for most of your public preview development needs.  public-preview-pnp branch is our advance working branch which might contains breaking changes.**
```

Comment:
Nits: I would remove "most of".

And at end of sentence, something like "... your public preview development needs, unless you have been directed to be on public-preview-pnp for a specific purpose".